openshift/openshift-azure
# Fix Azure Monitor Cluster UX Tab

This fixes the ARO cluster view in the Azure Monitor UX. This fix was decided on after a debugging session with @rashmichandrashekar.

See the `charles-mon` cluster in the screenshots below.

```release-note
Fix azure monitor cluster view user experience
```

**High-Level Cluster View**
![highlevel](https://user-images.githubusercontent.com/5978370/64726874-d64df080-d4d7-11e9-81ab-b852aa5da154.png)

**Specific Cluster View (charles-mon)**
![charles-mon](https://user-images.githubusercontent.com/5978370/64726932-f8477300-d4d7-11e9-8c86-3d4f49a6c8ec.png)


To verify these for yourself:

- Go to https://aka.ms/ci-privatepreview
- Change `Cloud Type: Azure` to `Cloud Type: All` in the dropdown menu at the top left side of the page to view the high-level ARO clusters known to Azure Monitor.
- The `charles-mon` entry on that page displays high-level cluster details such as kubernetes version, number of pods.
- The other entries do not show such high-level cluster details.
- Select `charles-mon` to see the case where the specific cluster UX is working and showing graphs
- Select any of the other clusters to see the case where the specific cluster UX is broken

/cc @asalkeld @ehashman @Makdaam @thekad
